### DISCUSSION TITLE
Incorrect Username/Password

### DISCUSSION THREAD

**Discussion by brooksduggan** [tayler6000/pyVoIP#166] [Q&A]
I have been scratching my head a lot over this, because I know my credentials are correct - so maybe it's something I don't see/understand.

I have a SIP domain with Twilio that works with both Zoiper and Blink softphones, but I am unable to successfully log into it with the example.
![image](https://github.com/tayler6000/pyVoIP/assets/31492302/13a6e6db-5ec9-4a18-9f10-89f5441ae70a)

Here is what I'm using (generally):
`# ACCOUNT DATA:`
`SIP_IP = "523414325.sip.us1.twilio.com"`
`SIP_Port = 5060`
`SIP_Username = "+19493042933"`
`SIP_Password = "testing1234"`
`myIP = "placeholder"`

Any ideas why it's not able to work via this the pyVoIP package, but works with the other soft phones?

**Comment by dmittal77** [comment]
Hi, have you been able to resolve this ? 
It still doesn't register with Twilio SIP.
